Libplanet.Explorer.Executable/Program.cs

```diff
@@ -40,11 +41,33 @@ public static async Task Main(string[] args)
                 .WriteTo.Console();
             Log.Logger = loggerConfig.CreateLogger();
 
-            IStore store = new DefaultStore(
-                path: options.StorePath,
-                flush: false,
-                readOnly: options.Seeds is null
-            );
+            IStore store;
+            if (options.Seeds.Any())
+            {
+                // Initialized DefaultStore.
+                store = new DefaultStore(
+                    path: options.StorePath,
+                    flush: false,
+                    readOnly: options.Seeds is null
+                );
+
+                // Warp up store.
+                store = new RichStore(
+                    store,
+                    path: options.StorePath,
+                    flush: false,
+                    readOnly: options.Seeds is null
+                );
+            }
+            else
+            {
+                store = new DefaultStore(
+                    path: options.StorePath,
+                    flush: false,
+                    readOnly: options.Seeds is null
+                );
```

Comment:
This seems equivalent to the first statement in the above clause.  Could you merge these two statements into a single statement and extract it from the `if`–`else` clauses?